.github/workflows/check-release.yml

```diff
@@ -0,0 +1,30 @@
+name: Check Release
+on:
+  push:
+    branches: ["master"]
+  pull_request:
+    branches: ["*"]
+
+jobs:
+  check_release:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        group: [check_release, link_check]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Base Setup
+        uses: jupyterlab/maintainer-tools/.github/actions/base-setup@v1
+      - name: Install Dependencies
+        run: |
+          pip install -e .
+      - name: Check Release
+        if: ${{ matrix.group == 'check_release' }}
+        uses: jupyter-server/jupyter_releaser/.github/actions/check-release@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Run Link Check
+        if: ${{ matrix.group == 'link_check' }}
+        uses: jupyter-server/jupyter_releaser/.github/actions/check-links@v1
```

Comment:
Maybe we can add the additional step that uploads the built assets as artifacts. This makes it easier to troubleshoot potential issues, and make it possible to double check everything looks good manually if needed.

Similar to: https://github.com/jupyterlab/jupyterlab/blob/a41b9816fdedb936be1589dbc98363f62f5f4574/.github/workflows/check-release.yml#L37-L43

```yaml
- name: Upload Assets
  if: ${{ matrix.group == 'check_release' }}
  uses: actions/upload-artifact@v2
  with:
    name: jupyterlab-releaser-dist-${{ github.run_number }}
    path: |
      .jupyter_releaser_checkout/dist/*.*
```

Reply:
:+1: